packages/wallets/src/components/WalletTourGuide/WalletTourGuideSettings.tsx

```diff
@@ -50,5 +50,5 @@ export const SpotLightHeader = ({ children }: PropsWithChildren) => (
 );
 
 export const getFiatWalletLoginId = (wallets?: THooks.WalletAccountsList[]) => {
-    return wallets?.find(wallet => !wallet.is_crypto)?.loginid;
+    return wallets?.find(wallet => typeof wallet.is_crypto === 'boolean' && !wallet.is_crypto)?.loginid;
```

Comment:
The `is_crypto` property is populated when currency config is loaded, while when it's still loading each wallet in wallet list has `is_crypto === undefined`, which this function previously evaluated as being a fiat wallet